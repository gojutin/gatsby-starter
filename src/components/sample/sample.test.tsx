import React from "react";
import Sample from "./sample";
import { render, cleanup } from "@testing-library/react";

afterEach(cleanup);

it("matches snapshot", () => {
  const { asFragment } = render(<Sample name="Bob" />);
  expect(asFragment()).toMatchSnapshot();
});
