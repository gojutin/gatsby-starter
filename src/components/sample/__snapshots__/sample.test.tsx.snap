// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <h3>
    Hello Bob
  </h3>
</DocumentFragment>
`;
